lib/core/src/sdk.rs

```diff
@@ -2105,6 +2107,7 @@ impl LiquidSdk {
                 };
 
                 let address = self.onchain_wallet.next_unused_address().await?.to_string();
+                println!("address: {address}");
```

Comment:
Can remove this I guess

Reply:
Yes, thanks. Went in by mistake. [0654cda](https://github.com/breez/breez-sdk-liquid/pull/707/commits/0654cda8331f4cd1ab0ac3a63ed615ea02012d35)